src/js/get_profile_data.js

```diff
@@ -16,16 +16,16 @@
   const dashboardRelayAliasCards = document.querySelectorAll("[data-relay-address]");
   const relayAddresses = [];
 
-   //Get FXA dashboard data from profile
-   const {aliasesUsedVal, emailsForwardedVal, emailsBlockedVal} = document.querySelector("firefox-private-relay-addon-dashboard-data").dataset;
-   browser.storage.local.set({aliasesUsedVal, emailsForwardedVal, emailsBlockedVal});
-   
    // Get FXA Stuff
    const fxaSubscriptionsUrl = document.querySelector("firefox-private-relay-addon-data").dataset.fxaSubscriptionsUrl;
    const premiumProdId = document.querySelector("firefox-private-relay-addon-data").dataset.premiumProdId;
    const premiumPriceId = document.querySelector("firefox-private-relay-addon-data").dataset.premiumPriceId;
+   const aliasesUsedVal = document.querySelector("firefox-private-relay-addon-data").dataset.aliasesUsedVal;
+   const emailsForwardedVal = document.querySelector("firefox-private-relay-addon-data").dataset.emailsForwardedVal;
+   const emailsBlockedVal = document.querySelector("firefox-private-relay-addon-data").dataset.emailsBlockedVal;
    const premiumSubdomainSet = document.querySelector("firefox-private-relay-addon-data").dataset.premiumSubdomainSet;
-   browser.storage.local.set({fxaSubscriptionsUrl, premiumProdId, premiumPriceId, premiumSubdomainSet});
+   const premiumEnabled = document.querySelector("firefox-private-relay-addon-data").dataset.premiumEnabled;
+   browser.storage.local.set({fxaSubscriptionsUrl, premiumProdId, premiumPriceId, aliasesUsedVal, emailsForwardedVal, emailsBlockedVal, premiumSubdomainSet, premiumEnabled});
```

Comment:
nit: make this more readable since there are so many items. 
```suggestion
   browser.storage.local.set({
    fxaSubscriptionsUrl,
    premiumProdId,
    premiumPriceId,
    aliasesUsedVal,
    emailsForwardedVal,
    emailsBlockedVal,
    premiumSubdomainSet,
    premiumEnabled
   });
```

Reply:
Done!